src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -3931,7 +3931,7 @@ public void testCreateBankAccountStatusTransition_noUserToken() {
     }
 
     @Test
-    public void testCreateBankAccountStatusTransition_noBankAccountToken() {
+    public void     testCreateBankAccountStatusTransition_noBankAccountToken() {
```

Comment:
```suggestion
    public void testCreateBankAccountStatusTransition_noBankAccountToken() {
```
Just to remove extra spacing

Reply:
@wmews-hw My bad. Removed the extra space.